### PR TITLE
Add movie command.

### PR DESCRIPTION
**Pull request by InATrance**
New movie command.

### CODE REVIEW

File: src/commands/Movie.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	async execute({ message, args }) {
+		const MovieDB = require('movieDB')(this.config.movieDBKey);
+		args = args.join(' ');
+
+		MovieDB.searchMovie({
+			query: args,
+			page: 1,
+			include_adult: false
+		}, (err, res) => {
+			if (err) return await this.error(msg.channel, 'Uh oh! Something went wrong with that query!');
```
Comment (briantanner) [inline src/commands/Movie.js]: This isn't an async function, await would throw

File: src/commands/Movie.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+			res = res.results[0];
+
+			let relDate = (new Date(res.release_date)).toLocaleDateString('en-US', {
+				weekday: 'long',
+				year: 'numeric',
+				month: 'long',
+				day: 'numeric'
+			});
+
+			return await this.sendMessage(msg.channel, {
```
Comment (briantanner) [inline src/commands/Movie.js]: This shouldn't return await, the command has to return a Promise, returning `this.sendMessage()` would be good.

File: src/commands/Movie.js (hunk 1 above)
Comment (briantanner) [reply src/commands/Movie.js]: Also this return is in the callback, and could result in the command returning void which would error in the bot.

File: src/commands/Movie.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        this.module         = 'Fun';
+        this.description    = 'Get info on a movie.';
+        this.usage          = 'movie [movie name]';
+        this.example        = 'movie Deadpool';
+        this.cooldown       = 3000;
+        this.expectedArgs   = 1;
+	}
+
+	async execute({ message, args }) {
+		const MovieDB = require('movieDB')(this.config.movieDBKey);
```
Comment (briantanner) [inline src/commands/Movie.js]: Should be at the top of the file, or we'll be requiring it every time the command is run, can instantiate it in the constructor as a property prefixed by `_`
```
const movieDB = require('movieDB');
...
constructor() {
  this._movieDB = movieDB(this.config.movieDBKey);
}
```

**Comment by InATrance** [comment]
👍

**Comment by briantanner** [comment]
One last thing, forgot to npm install the moviedb module

**Comment by InATrance** [comment]
Added dependency and fixed merge conflicts.

**Pull request merged**